css/styles.css

```diff
@@ -0,0 +1,217 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 960px;
+  margin: 0 auto;
+  overflow: auto;
+  font-family: "Calibri", sans-serif;
+}
+
+p {
+  font-size: 18pt;
+}
+
+h1 {
+  color: yellow;
+  font-family: serif;
+  padding-left: 20px;
+  font-size: 28pt;
+  letter-spacing: 0.05em;
+}
+
+h2 {
+    font-size: 24pt;
+    font-family: serif;
+    text-align: center;
+}
+
+h3 {
+    font-size: 26pt;
+    font-family: serif;
+    text-align: left;
+    letter-spacing: 0.05em;
+    word-spacing: 160%;
+    margin-bottom: 0px;
+}
+
+h4 {
+    font-size: 24pt;
+    font-family: "Calibri" sans-serif; 
+    text-align: left;
+    letter-spacing: 0.04em;
+    margin-bottom: 0px;
+}
+
+header, footer {
+  width: 100%;
+  height: 100%;
+  background-color: black;
+}
+
+header {
+  height: 148px;
+  overflow: auto;
+  border-bottom: solid yellow 4px;
+}
+
+nav {
+  width: 100%;
+  background-color: black;
+  padding-right: 5%;
+  display: block;
+  text-align: right;
+  color: white; 
+  font-size: 16pt;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+  padding-left: 4px;
+  padding-right: 4px;
+  justify-content: space-around;
```

Comment:
@jbistis You can remove this rule, this is used only within `flex` and `grid` container' 🔨